templates/zerver/help/enable-or-disable-pressing-enter-to-send.md

```diff
@@ -1,21 +1,16 @@
-# Press Enter to send
+# Enable or disable Press Enter to send
 
 By default, Zulip sends your message when you press enter (or return)
-on your keyboard. If you want to turn this off, simply uncheck the
-**Press Enter to send** box below the compose field:
+on your keyboard.
 
-![Box checked](/static/images/help/enter-box-checked.png)
+If you want to disable this feature, simply uncheck the
+**Press Enter to send** option below the compose box.
```

Comment:
Should we point the user to the bottom right corner since the screenshot has been removed?